vnext/Desktop.DLL/pch.cpp

```diff
@@ -2,3 +2,29 @@
 // Licensed under the MIT License.
 
 #include "pch.h"
+
+#include <RuntimeOptions.h>
```

Comment:
This is the wrong file to put any code in. Th purpose of thius file must be only to generate the pch file. Please create a new .cpp file. E.g. DllMain or DelayLoadHook or something else along these lines.